frontend/src/fixtures/articleFixtures.js

```diff
@@ -0,0 +1,42 @@
+const articleFixtures = {
+  oneArticle: {
+    id: 1,
+    title: "Title One",
+    url: "https://example.com/",
+    explanation:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    email: "whatever@mailinator.com",
+    dateAdded: "2022-01-02T12:00:00",
+  },
+  threeArticles: [
+    {
+      id: 1,
+      title: "Title One",
+      url: "https://example.com/",
+      explanation:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+      email: "whatever@mailinator.com",
```

Comment:
nit: maybe have more variation in the values, the mutation and tests might not work properly with the same info copy-pasted three times